client/web/app/elements/lecture/view.html

```diff
@@ -150,8 +122,8 @@
             },
             ready: function(){
                 console.log("width: ", window.innerWidth);
-                if(window.innerWidth > 1279) this.videoHeight = "480px";
-                else if(window.innerWidth > 799) this.videoHeight = "300px";
+                if(window.innerWidth > 1279) this.videoHeight = 480;
+                else if(window.innerWidth > 799) this.videoHeight = 300;
                 console.log("height: ", this.videoHeight);
```

Comment:
mysle ze jak cos piszemy to usuwajmy niektore logi bo sa czasem bez sensu tak jak slabe jest tutaj wypisywanie szerokosci i wysokosci/ co moim zdaniem nam nic nie mowi. Trzeba je moim zdaniem uczytelnic lub wywalac